NuclearData/ceNeutronData/ceNeutronCache_mod.f90

```diff
@@ -33,14 +33,19 @@ module ceNeutronCache_mod
   !!   E_tail -> Energy of all XSs in xss except total
   !!   f      -> Interpolation factor for the nuclide at energy E_tot
   !!   idx    -> Index on a nuclide grid for energy E_tot
-  !!   xss    -> Cached Cross-Section values
+  !!   xss    -> Cached cross-section values
+  !!   E_rel  -> Base energy for which relative energy cross sections are found (for TMS)
+  !!   xssRel -> Cached effective cross-section values at energy relative to E_rel (for TMS)
```

Comment:
> The XSs in `xssRel` are cross-sections that contain contributions from other nuclides in the mixture at different relative energies?

This is exactly it. I wanted to store those instead of resampling for when you need to score multiple tallies per each collision. One could resample and that would be equally correct, but much slower. So it depends on what we prioritise.

So E_rel is again the energy of the particle in lab frame, but those xssRel are different from the material xss(E_rel). So again having two flags is necessary.